doc/scripts/README.md

```diff
@@ -932,7 +932,7 @@ b. Внутри литерала массива после `[` и перед `]`
    }
```

Comment:
These changes will be relevant for the version README.ru.md. We need not to get confused and have all the current changes in only one file.

Reply:
Done)